### PR TITLE
docs: update posts with multiple options in app

### PR DESCRIPTION
**Pull request by thlmenezes**
## Description of change

- Update posts that document routes in the web app with multiple options, e.g

1. Add a Server
2. Add a Credential
3. List Environments

- Removes draft flag from posts related to features already in production.
- Updates the getting started guide link from `docs/servers/index.md` to `docs/servers/add-server.md`

## Issues resolved by this PR

- [x] DVN-11650
- [x] DVN-11651
- [x] DVN-11652
- [x] DVN-11653

### CODE REVIEW

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #730 (docs: update posts with multiple options in app)`: Devopness application `thlmenezes-site.devopness.com` deployment **completed** on action <https://thlmenezes-site.devopness.com>

File: docs/environments/list-environments.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: List Environments
+intro: Access the list of project environment to see details of an environment.
```
Comment (jfoliveira) [inline docs/environments/list-environments.md:3]:
```suggestion
intro: You can find and view details of project environments you have access to
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

**Comment by jfoliveira** [comment]
> Update posts that document routes in the web app with multiple options, e.g
> 
> 1. Add a Server

@thlmenezes I don't see any changes in this PR related to the `Add a server` workflow. Are there missing commits in the PR branch? 🤔

File: docs/environments/list-environments.md (hunk 1 above)
Comment (thlmenezes) [reply docs/environments/list-environments.md]: intro updated to better describe the post objective

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #730 (docs: update posts with multiple options in app)`: Devopness application `thlmenezes-site.devopness.com` deployment **in-progress** on action <https://thlmenezes-site.devopness.com>

**Comment by thlmenezes** [comment]
> @thlmenezes I don't see any changes in this PR related to the `Add a server` workflow. Are there missing commits in the PR branch? 🤔

You are correct, there were missing changes.

PR updated

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #730 (docs: update posts with multiple options in app)`: Devopness application `thlmenezes-site.devopness.com` deployment **completed** on action <https://thlmenezes-site.devopness.com>

**Review by jfoliveira — APPROVED** [review]

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #730 (docs: update posts with multiple options in app)`: Devopness application `thlmenezes-site.devopness.com` deployment **in-progress** on action <https://thlmenezes-site.devopness.com>

**Comment by thlmenezes** [comment]
Deployed pipeline for `PR #730 (docs: update posts with multiple options in app)`: Devopness application `thlmenezes-site.devopness.com` deployment **failed** on action <https://thlmenezes-site.devopness.com>

**Pull request merged**